docs/platform/tutorials/currency-alerting.md

```diff
@@ -94,112 +57,186 @@ Let’s set up each function one by one, starting with our input function.
 
-1. 
-Ensure the input topic is "coin-data".
+**Purpose:** _Send a push notification when certain conditions are met_
 
+This function reads from the `currency-rate-alerts` topic and whenever a new message arrives, it sends a push notification to the Pushover app on your mobile phone.
 
-2. 
-"Numbers" should be the command separated list of phone numbers to send SMS alerts to.
+It also reads the contents of the message and enriches the notification with details on how the threshold was crossed (specifically, if the price is moving up or down).
 
+**To set up the push nofication function, follow these steps:**
 
-3. 
-The account_sid, auth_token and messaging_service_sid can be populated using the values you saved while setting up your Twilio account.
+1. Open the Quix Library one last time.
+   <br><br>
+2. In the search box on the library page, search for "_Pushover_".<br>
+   You should see a result that resembles the following example: ![Pushover Notifications](../images/tutorials/currency-alerting/library-pushover.png "Pushover Notifications")
+   <br><br>
+3. Click **Preview Code**, and on the page that appears, click **Edit Code**.
+   <br><br>
 
+4. On the 'Project Creation' page, fill out the following fields:
+   
+    | Field      | Value                          |
+    | ----------- | ------------------------------------ |
+    | **Name:**      | As usual, enter a project name or keep the default suggestion. |
+    | **Input:**      | Select the input topic—in this case, select `currency-rate-alerts` from the list.<br>We're going to be reading every message from this topic and turning it into a push notification. |
+    | **base_url:**    | Leave the default value, `https://api.pushover.net/1/messages.json?` (if you decide to use another push notification app, your can always update this value). |
+    | **api_token:**    | Enter the API token that you received when you signed up with Pushover (for example: `azovmnbxxdxkj7j4g4wxxxdwf12xx4`) |
+    | **user_key:**    | Enter the user key that you received when you signed up with Pushover (for example: `u721txxxgmvuy5dxaxxxpzx5xxxx9e`) |
 
-4. 
-message_limit can be left at 2 or changed to suit your needs.
-Click Deploy and wait for the pre-built connector to be depolyed to your workspace.
+5. Click **Save as Project**.<br>
+   You now have a copy of the boilerplate Pushover notification function in your workspace
+   <br><br>
+6. For the last time, double check that the environment variables are correct, then click **Deploy**.
 
-**Logs**
+**Checking to see if it worked**
 
-Once deployed you will again be redirected to the workspace home page. You can click on the Twilio Sink tile and view the logs if you wish.
+One obvious sign that it work is that you'll start getting Pushover notifications on your phone that resemble the following example:
+![Pushover Notification Example](../images/tutorials/currency-alerting/pushover_notification.png)
```

Comment:
I think the pushover image is too large.

Reply:
I've resized to 60%. Looks much better.